ui/pages/ss.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useMemo, useState, useRef } from "react";
+import {
+  Box,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from "@material-ui/core";
+import { gql, useQuery } from "urql";
+import HappySpinner from "components/HappySpinner";
+import LoadMore from "components/LoadMore";
+import dayjs from "dayjs";
+import { toMS } from "utils/date";
+
+const GET_SUPER_ADMIN_SESSIONS = gql`
+  query GetSuperAdminSessions($limit: Int!, $offset: Int!) {
+    getSuperAdminSessions(limit: $limit, offset: $offset) {
+      moreExist
+      sessions {
+        id
+        start
+        end
+        duration
+        user {
+          username
+        }
+      }
+    }
+  }
+`;
+
+function SS() {
+  const limit = 15;
+  const [offset, setOffset] = useState(0);
+  const [{ data, fetching }] = useQuery({
+    query: GET_SUPER_ADMIN_SESSIONS,
+    variables: { offset, limit },
+  });
+  const moreExist = data?.getSuperAdminSessions?.moreExist;
+  const offsetRef = useRef(0);
+  const sessionsRef = useRef([]);
+
+  const sessions = useMemo(() => {
+    if (data?.getSuperAdminSessions.sessions) {
+      offsetRef.current = offset;
+      sessionsRef.current = sessionsRef.current.concat(
+        data?.getSuperAdminSessions.sessions || []
+      );
+      return sessionsRef.current;
+    }
+  }, [data?.getSuperAdminSessions.sessions, offset]);
+
+  if (fetching && (sessions?.length === 0 || !sessions))
+    return <HappySpinner />;
+
+  return (
+    <div className="page">
+      <div className="bg-white rounded-lg shadow overflow-hidden">
+        <TableContainer>
+          <Table aria-label="simple table">
+            <TableHead>
+              <TableRow>
+                <TableCell>Admin</TableCell>
+                <TableCell>Start</TableCell>
+                <TableCell>End</TableCell>
+                <TableCell>Duration</TableCell>
```

Comment:
The duration for the current session is a bit buggy, for a 15 min session it's listed as "15:00 / 15"